packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -229,8 +229,6 @@ const DropdownMenuRootContent = React.forwardRef((props, forwardedRef) => {
       onCloseAutoFocus={composeEventHandlers(props.onCloseAutoFocus, (event) => {
         event.preventDefault();
         if (!preventCloseAutoFocusRef.current) context.triggerRef.current?.focus();
-      })}
-      onEscapeKeyDown={composeEventHandlers(props.onEscapeKeyDown, () => {
```

Comment:
We need to reset the ref here between closes, this change previously assumed it would only ever be closed via `escape` key. Now it will reset after every close regardless.

Reply:
This is one of those cases where we need to make sure they can't cancel that bit of the behaviour then right? Because right now, if they prevent default to run their own focus, the ref won't be reset.

Reply:
I think that depends, the current assumption makes this moot because if we `preventDefault` on `onCloseAutoFocus` then we're opting out of the built in focus handling all together right? or is the expectation that the user is cancelling the default behaviour of **the handler**, and that the conditions with which it is run should persist? e.g. the `onCloseAutoFocus` handler itself is subject to pointer conditions, not just the focus behaviour within it.

Reply:
Well I think it's more about ensuring the "integrity" of the system. That's kinda when we need to think about which bit they can or cannot prevent. ie. here it's important that it gets reset properly when it's actually closed.